pg_bm25/src/api/bootstrap.rs

```diff
@@ -0,0 +1,11 @@
+use pgrx::*;
+use std::fs;
+use std::path::PathBuf;
+
+#[pg_extern]
+pub fn create_bm25_test_table() {
+    let base_path = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+    let sql_file_path = base_path.join("sql/_bootstrap_quickstart.sql");
+    let file_content = fs::read_to_string(sql_file_path).expect("Error reading SQL file");
+    let _ = Spi::run_with_args(&file_content, None);
```

Comment:
You can use `Spi::run(&file_content)` if you don't actually have any args you need to pass.

I'd suggest adding an `.expect("could not create bm25_test_table")` to the `Result` so we have a little extra context if this blows up.

Reply:
done